src/Meshes/mesh.ts

```diff
@@ -71,14 +71,17 @@ class _InstanceDataStorage {
     public batchCache = new _InstancesBatch();
     public instancesBufferSize = 32 * 16 * 4; // let's start with a maximum of 32 instances
     public instancesBuffer: Nullable<Buffer>;
+    public instancesPreviousBuffer: Nullable<Buffer>;

```

Comment:
So in order to not go over the 128 properties per object (which triggers the slow path in v8, please add all these new properties in the _instanceDataStorage